package.json

```diff
@@ -25,73 +25,78 @@
+    "@babel/plugin-transform-runtime": "^7.11.5",
+    "@babel/preset-env": "^7.11.5",
+    "@babel/preset-react": "^7.10.4",
+    "@emotion/core": "^10.0.35",
     "@emotion/styled": "^10.0.27",
-    "@leafygreen-ui/checkbox": "^4.0.1",
-    "@leafygreen-ui/code": "^3.4.4",
+    "@leafygreen-ui/checkbox": "^4.1.1",
+    "@leafygreen-ui/code": "^3.5.0",
     "@testing-library/react-hooks": "^3.4.1",
     "copy-to-clipboard": "^3.3.1",
-    "css-loader": "^3.5.3",
+    "css-loader": "^4.3.0",
     "dlv": "^1.1.3",
     "dotenv": "^8.2.0",
     "emotion-theming": "^10.0.27",
     "esm": "^3.2.25",
-    "fast-xml-parser": "^3.17.3",
-    "gatsby": "^2.22.11",
-    "gatsby-plugin-emotion": "^4.3.2",
-    "gatsby-plugin-feed": "^2.5.3",
-    "gatsby-plugin-google-tagmanager": "^2.3.3",
+    "fast-xml-parser": "^3.17.4",
+    "gatsby": "^2.24.54",
+    "gatsby-plugin-emotion": "^4.3.10",
+    "gatsby-plugin-feed": "^2.5.11",
+    "gatsby-plugin-google-tagmanager": "^2.3.11",
     "gatsby-plugin-meta-redirect": "^1.1.1",
-    "gatsby-plugin-react-helmet": "^3.3.2",
-    "gatsby-plugin-sitemap": "^2.4.3",
+    "gatsby-plugin-react-helmet": "^3.3.10",
+    "gatsby-plugin-sitemap": "^2.4.12",
     "memoizerific": "^1.11.3",
     "mongodb-stitch-browser-sdk": "^4.8.0",
     "mongodb-stitch-server-sdk": "^4.8.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-aria-modal": "^4.0.0",
     "react-dom": "^16.13.1",
-    "react-helmet": "^6.0.0",
+    "react-helmet": "^6.1.0",
     "react-loadable": "^5.5.0",
-    "react-player": "^1.15.3",
-    "react-tiny-popover": "^5.0.1",
+    "react-player": "^2.6.1",
+    "react-refresh": "^0.8.3",
+    "react-tiny-popover": "^5.1.0",
     "style-loader": "^1.2.1",
     "typeface-fira-mono": "^0.0.72",
-    "webpack": "^4.43.0",
-    "webpack-cli": "^3.3.11",
+    "typescript": "^4.0.2",
+    "webpack": "^4.44.1",
+    "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0",
-    "whatwg-fetch": "^3.0.0"
+    "whatwg-fetch": "^3.4.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.10.0",
+    "@babel/core": "^7.11.6",
     "@emotion/babel-preset-css-prop": "^10.0.27",
-    "@storybook/addon-actions": "^5.3.19",
-    "@storybook/addon-docs": "^5.3.19",
-    "@storybook/addon-knobs": "^5.3.19",
-    "@storybook/react": "^5.3.19",
-    "babel-jest": "^26.0.1",
+    "@storybook/addon-actions": "^6.0.21",
+    "@storybook/addon-docs": "^6.0.21",
+    "@storybook/addon-knobs": "^6.0.21",
+    "@storybook/react": "^6.0.21",
+    "acorn": "^7.4.0",
+    "babel-jest": "^26.3.0",
     "babel-loader": "^8.1.0",
     "babel-plugin-emotion": "^10.0.33",
-    "babel-preset-gatsby": "^0.4.7",
-    "cypress": "^4.7.0",
+    "babel-preset-gatsby": "^0.5.9",
+    "core-js": "^3.6.5",
+    "cypress": "^5.1.0",
     "enzyme": "^3.11.0",
-    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-adapter-react-16": "^1.15.4",
     "enzyme-to-json": "^3.5.0",
-    "eslint": "^7.1.0",
```

Comment:
For example, three of the remaining seven warnings involve `eslint`:

```
npm WARN @typescript-eslint/eslint-plugin@2.34.0 requires a peer of eslint@^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @typescript-eslint/parser@2.34.0 requires a peer of eslint@^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-config-react-app@5.2.1 requires a peer of eslint@6.x but none is installed. You must install peer dependencies yourself.
```

Instead of downgrading `eslint` on our end, I think it makes more sense to wait for gatsby to upgrade its dependencies and then just upgrade gatsby again when it is ready.

The other warnings are similar for gatsby, react, and react-dom. Just checking out these packages, they still function well and newer versions have (mostly) bumped their peer dependency requirements